pyaerocom/aeroval/setupclasses.py

```
@@ -88,15 +96,12 @@ def get_json_output_dirs(self, assert_exists=True):
         return out
 
 
-class ModelMapsSetup(ConstrainedContainer):
-    maps_freq = EitherOf(["monthly", "yearly"])
+class ModelMapsSetup(BaseModel):
+    maps_freq: Literal["monthly", "yearly"] = "monthly"
+    maps_res_deg: PositiveInt = 5
```

The original class supported `kwargs`, was that feature redundant?

---

As far as I can tell, yes. Moreover, we don't want these classes taking keyword arguments which aren't defined. Part of this rewrite is to rigorously define the configuration interface, so passing arguments which are not present should raise an error.